project_stock/models/project_task.py

```diff
@@ -62,7 +62,7 @@ class ProjectTask(models.Model):
         index=True,
         check_company=True,
     )
-    stock_analytic_date = fields.Date(string="Analytic date")
+    stock_analytic_date = fields.Date(string="Stock Analytic date")
```

Comment:
Doesn't precommit warn you that it is redundant?

Reply:
Oops, pre-commit does not give error because the string is not “Stock Analytic **D**ate”, actually that would be the right one, so I will remove the string.

Reply:
But please change translations then.